app/uk/gov/hmrc/securemessage/controllers/binders/package.scala

```diff
@@ -49,8 +49,8 @@ package object binders {
 
       override def bind(key: String, params: Map[String, Seq[String]]): Option[Either[String, Language]] =
         stringBinder.bind("lang", params) map {
-          case Right(language) => Right(Language.namesToValuesMap.getOrElse(language, English))
```

Comment:
@kjainsc I guess this change to support both the cases of CY, but instead of explicitly doing check,  better to do `language.toLowerCase` , that way it will be more simpler I believe..

Reply:
I was thinking to include check for only specific `cy` & `CY` params but I think it does not need to be that strict. I have updated the pr. thanks